src/test/resources/org/prebid/server/it/openrtb2/bizzclick/test-auction-bizzclick-request.json

```diff
@@ -9,6 +9,7 @@
       },
       "ext": {
         "bizzclick": {
+          "host": "host",
           "accountId": "accountId",
           "placementId": "placementId"
```

Comment:
I suggest tweaking the BizzclickTest and the configured endpoint url accordingly to check whether these macros are really resolved